cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -215,15 +216,21 @@ public BodyConsumer createAppVersion(HttpRequest request, HttpResponder responde
   }
 
   /**
-   * Returns a list of applications associated with a namespace.
+   * Returns a list of {@link ApplicationRecord} for all applications in the given namespace
+   *
+   * @param request         {@link HttpRequest}
```

Comment:
style: our javadocs usually aren't aligned and just use a single space between param and description

Reply:
hm..the cask setting I imported set para aligned in the javadoc. 

Disabled and reformatted the code.